0008-quilt-ethos.md

```diff
@@ -0,0 +1,46 @@
+# Summary
+
+Quilt has a set of Community Guidelines and Governance RFCs. These provide a set of rules, expectations and guidance on what and how certain aspects of the project should be managed. After some reflection, it seems that a set of "Values" had not been define; the aim of this document is to set these values.
+
+# Motivation
+
+This can be seen as a derivate or inspiration from the Python Projects [PEP20](https://www.python.org/dev/peps/pep-0020/). I (ToffeeMax) have found this document useful when trying to both write better code using Python and as a general set of development principles. PEP20 could partially be seen as non-serious, however it is referenced within Python frequently. 
+
+The can be refered back to as more of a "Why" a decision was made - instead of the rigid proccess of HOW that other RFCs intend.
+
+
+# Explanation
+
+We will sometimes get things wrong, but:
+Do not let perfect be the enemy of actually shipping an idea.
+Modular, Simple and Flexible.
+Made by Anyone;
+Made for Everyone.
+This project breaths on collaboration.
+And colloboration will shape this into something amazing.
+Not sure whether something will work? Try it. 
+Spread out power and responsibilty; it is a heavy burden to walk alone.
+Tiny Potato got old; find an new hill to march apon.
+I hear a pineapple is good?
+
+
+# Drawbacks
+
+It could be used in future as a "hill to die on" and be constantly refered back to in reference to why a certiain decision has been made or those seeking to question a decision as not being inline with the project
```

Comment:
```suggestion
It could be used in future as a "hill to die on" and be constantly referred back to in reference to why a certain decision has been made or those seeking to question a decision as not being inline with the project
```

Typos